docs/v3/source/includes/resources/builds/_create.md.erb

```diff
@@ -51,4 +51,5 @@ Name | Type | Description | Default
 --- | ---
 Admin |
 Space Developer |
+Space Suporter | Experimental
```

Comment:
Super minor but there's a typo in "Supporter", otherwise these changes look good, thanks!